website/docusaurus.config.js

```diff
@@ -267,7 +267,7 @@ module.exports = {
         },
         announcementBar: {
             id: `crawlee-for-python-v1`,
```

Comment:
you need to change the ID too, otherwise it would stay hidden for people who already saw the old ID